### PR TITLE
Don't use tot any more

### PR DESCRIPTION
**Pull request by Katharine**
Stop using tot. This leaves tot running (for now), but stops actually referencing it, so we will default to snowflake IDs.

/hold

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Is there a migration doc/paragraph on what someone who's using `tot` today would need to do to stop using it?

**Comment by Katharine** [comment]
We don't really have a document. To answer inline: no part of prow "proper" particularly cares whether you do or do not use tot (so, in that respect, just "be up to date"). Sufficiently recent Gubernator also does not care whether you use tot. Kettle supports not using tot, but needs to be explicitly told this (e.g. #11895). Testgrid doesn't care how builds are numbered, as long as they are monotonic (so switching back to tot is problematic).

So basically just removing the URL references and (technically optionally) turning down the deployment should do the job. The hard part is switching _to_ tot.

**Comment by stevekuznetsov** [comment]
Sounds good. If you deprecate `tot` we should put that blurb in ANNOUNCEMENTS. Sounds like few people will have a difficult transition

**Comment by spiffxp** [comment]
/lgtm
Release is out the door.  I leave it to @Katharine to `/hold cancel` as desired
